ApollosProject/apollos-apps
# feat: dark mode

## DESCRIPTION
This PR adds dark mode support.

### What does this PR do, or why is it needed?
This PR adds native support for dark mode via the `useColorScheme` hook. This hook provides and subscribes to color scheme updates by exposing information about a user's appearance preferences, such as their preferred color scheme (light/dark).

### What design trade-offs/decisions were made?

### How do I test this PR?
To test this PR, change the settings to your iOS and Android device to dark. Open the app and confirm that the appearance is now in dark mode. Go back to settings and turn off dark preferences and reopen the app to see the color scheme change back to light.

|iOS|
|---|
|![Simulator Screen Shot - iPhone 11 Pro Max - 2020-12-18 at 12 21 20](https://user-images.githubusercontent.com/52924611/102642384-94c0e200-412b-11eb-91ed-310aa0795690.png)|
|![Simulator Screen Shot - iPhone 11 Pro Max - 2020-12-18 at 12 21 25](https://user-images.githubusercontent.com/52924611/102642389-968aa580-412b-11eb-8d15-43996fd1e2ba.png)|
|![Simulator Screen Shot - iPhone 11 Pro Max - 2020-12-18 at 12 21 23](https://user-images.githubusercontent.com/52924611/102642393-97bbd280-412b-11eb-82ce-65febbb0d309.png)|



### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
